pull/github.go

```diff
@@ -829,6 +834,89 @@ func (ghc *GitHubContext) getCheckStatuses() (map[string]string, error) {
 	return statuses, nil
 }
 
+func (ghc *GitHubContext) LatestWorkflowRuns() (map[string]string, error) {
+	if ghc.workflowRuns != nil {
+		return ghc.workflowRuns, nil
+	}
+
+	workflowRuns, err := ghc.getWorkflowRuns()
+	if err != nil {
+		return nil, err
+	}
+
+	ghc.workflowRuns = workflowRuns
+
+	return workflowRuns, nil
+}
+
+func (ghc *GitHubContext) getWorkflowRuns() (map[string]string, error) {
+	// TODO: We could use the REST API to get the path, once
+	// https://github.com/google/go-github/commit/7665f317d3985efe6a7828e3af2751c41b3526b1
+	// is in a release.
```

Comment:
Looking at the APIs, I think the REST API makes this more straightforward, so it might be worth copying the relevant function and the partial struct we need in to this project while waiting for a release.

https://github.com/palantir/policy-bot/blob/67b611ab74ebfd63f98addb6105e06a71e65bb7f/pull/github_teams.go is an example of where we did something similar previously when an API we still used was removed upstream.

Reply:
Since I left this comment, we upgraded to `go-github` v63, which should include the new REST API field you wanted to use originally.

Reply:
Just coming back to this. Yep, great! I'll switch over to REST. Cheers.